src/plugins/content_management/server/event_stream/es/integration_tests/es_event_stream_client.test.ts

```diff
@@ -0,0 +1,70 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0 and the Server Side Public License, v 1; you may not use this file except
+ * in compliance with, at your election, the Elastic License 2.0 or the Server
+ * Side Public License, v 1.
+ */
+
+import { ElasticsearchClient } from '@kbn/core/server';
+import {
+  createTestServers,
+  type TestElasticsearchUtils,
+  type TestKibanaUtils,
+} from '@kbn/core-test-helpers-kbn-server';
+import { EsEventStreamClient } from '../es_event_stream_client';
+import { EventStreamLoggerMock } from '../../tests/event_stream_logger_mock';
+import { testEventStreamClient } from '../../tests/test_event_stream_client';
+
+describe('EsEventStreamClient', () => {
+  let manageES: TestElasticsearchUtils;
+  let manageKbn: TestKibanaUtils;
+  let esClient: ElasticsearchClient;
+  let resolveClient: (client: EsEventStreamClient) => void = () => {};
+  const client: Promise<EsEventStreamClient> = new Promise((resolve) => {
+    resolveClient = resolve;
+  });
+
+  const baseName = '.kibana-test';
+  const indexTemplateName = `${baseName}-event-stream-template`;
+
+  beforeAll(async () => {
+    const { startES, startKibana } = createTestServers({ adjustTimeout: jest.setTimeout });
```

Comment:
I had no clue we could do this ... Nice!

Reply:
It has one downside, that it is very slow.